edge/cmd/edgecore/app/server.go

```diff
@@ -154,7 +154,8 @@ func environmentCheck() error {
 		case "kubelet": // if kubelet is running, return error
 			return errors.New("kubelet should not running on edge node when running edgecore")
 		case "kube-proxy": // if kube-proxy is running, return error
-			return errors.New("kube-proxy should not running on edge node when running edgecore")
```

Comment:
I think when user use edgemesh in the edge node, we had better not run kube-proxy

Reply:
Yes, now if users deploy edgemesh on edgenode, it should not install kube-proxy on edge, and now we can't support kube-proxy run normally by default for edge list-watch.

Reply:
It's one of the scenes. This check method may not be smooth and we can declare it in the Edgemesh guidebook instead.